master/getting-started/mesos/installation/docker.md

```diff
@@ -5,65 +5,92 @@ title: Installing Calico for the Docker Containerizer
 This document provides the commands to download and run Calico
 for use with the Docker Containerizer in Mesos.
 
-## Prerequisite: Docker
-Calico networks Docker tasks for Mesos with its libnetwork plugin. In order to
-run the libnetwork plugin, the Docker daemon on each agent must be configured
-with a cluster store.
+## Prerequisites
 
-If using etcd as a cluster store, for example, run the Docker daemon with the
-following additional parameter:
+#### 1. etcd
 
-```shell
-    --cluster-store=etcd://<ETCD HOST>:<PORT>
+Calico uses etcd as its datastore. Ensure you have an instance of etcd running,
+and that it is accessible from all Agents in your cluster.
+
+To maximize availability, we recommend running an etcd cluster across your mesos masters.
+See [etcd's clustering guide](https://coreos.com/os/docs/latest/cluster-architectures.html)
+for more information.
+
+For testing, you can quickly run a single instance of etcd using docker on your Master
```

Comment:
I'll remove it, but I intentionally put in "quickly" to imply that this is a fast hack, and the clustering is really the correct way to go in prod